formats/redirect/publisher/schema.json

```diff
@@ -52,7 +52,13 @@
   "definitions": {
     "absolute_path": {
       "type": "string",
-      "pattern": "^/(([a-zA-Z0-9._~!$&'()*+,;=:@-]|%[0-9a-fA-F]{2})+(/([a-zA-Z0-9._~!$&'()*+,;=:@-]|%[0-9a-fA-F]{2})*)*)?$"
+      "pattern": "^/(([a-zA-Z0-9._~!$&'()*+,;=:@-]|%[0-9a-fA-F]{2})+(/([a-zA-Z0-9._~!$&'()*+,;=:@-]|%[0-9a-fA-F]{2})*)*)?$",
+      "description": "A path only. Query string and/or fragment are not allowed."
+    },
+    "absolute_fullpath": {
```

Comment:
I agree. Naming of parts/combinations of parts of URLs is difficult. Here's where I got the idea for this name:
http://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-fullpath